### ISSUE TITLE
[Accessibility - multiple pages] Polymer logo alt text

### ISSUE THREAD

**Issue by lpalmaro** [Polymer/docs#1141]
Top left polymer icon should have alt text about Polymer logo, instead of "lock up image".

**Comment by arthurevans** [comment]
This image links back to the homepage (which is kind of a stupid link to have on the homepage, but that's a separate issue). 

I tested out a couple of approaches to this using VoiceOver. I can use "Polymer logo" as the alt text, in which case it says, "Visited link image polymer logo", which is correct but less helpful in predicting what the link does.

If I use "polymer home" as alt text, it says, "Visited link image polymer home", which is a little weird but suggests what the link does.

I tried adding aria-label="polymer home" to the anchor tag and specifying blank alt text for the image, but that didn't work (I got back to "svg lockup").

Any guidance on this? I'm inclined to go with "Polymer home."

**Comment by lpalmaro** [comment]
Good point -- if this is more than just an icon, this should certainly be
clear to the user. I also like "Polymer home", and if it's implemented as a
link, then the screen reader should tell the user that, as well.

**Comment by arthurevans** [comment]
Fixed in #1168.
